db/migrations/20221031120731_products.sql

```diff
@@ -0,0 +1,19 @@
+-- migrate:up
+CREATE TABLE products (
+    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    thumbnail VARCHAR(1000) NULL,
```

Comment:
썸네일 사진임이 명확해보이지만 이미지 url를 담는 컬럼인 것을 확실하게 구분지어 주면 좋을 것 같습니다.
또한 해당 숙소 사진은 서비스 이용자가 참고해야하는 자료 중 하나이므로 NULL 값을 허용하면 안될 것 같네요.

```suggestion
    thumbnail_url VARCHAR(1000) NOT NULL,
```